packages/go_router/example/README.md

```diff
@@ -0,0 +1,16 @@
+# Example Catalog
```

Comment:
Should we delete the rest for now? We can always revive them by going back through the git history.

Reply:
There are example that demonstrate advanced API, I will move them to a different folder for now. let me know if you think that is ok

Reply:
SGTM